pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -0,0 +1,173 @@
+package ovirt
```

Comment:
I would suggest we drop all code for terminal prompt for this new platform in this PR.

let's start with install-config.yaml file option only.

Reply:
Yes, this shouldn't be in the TUI until it's been tested by QE via install-config.yaml